lobid-vocab.ttl

```diff
@@ -256,9 +256,18 @@ lv:currentLocation
     rdfs:label "current location"@en ;
     rdfs:label "Standort"@de ;
     rdfs:isDefinedBy <http://purl.org/lobid/lv> ;
-    rdfs:range rdfs:Literal ;
-    rdfs:subPropertyOf dct:identifier .
+    rdfs:range rdfs:Literal .
 
+# <http://purl.org/lobid/lv#currentLibrary>
+
+lv:currentLibrary
+    a owl:DatatypeProperty ;
+    rdfs:comment "The specific library within an institution holding an item"@en ;
```

Comment:
```suggestion
    rdfs:comment "The code of specific library within an institution holding an item"@en ;
```

We might also add which kind of code this is. "Alma code"?

Reply:
```suggestion
    rdfs:comment "The code of the specific library within an institution holding an item used in ALMA"@en ;
```

Like this?